.github/workflows/workflow.yaml

```diff
@@ -58,40 +60,6 @@ jobs:
           micromamba run -n modyn pylint --version
           micromamba run -n modyn pylint modyn
 
-  isort:
```

Comment:
just to be sure, ruff replaces isort as well?

Reply:
Yes, https://docs.astral.sh/ruff/rules/#isort-i